src/app/Board/index.ts

```diff
@@ -0,0 +1,53 @@
+import './board.css'
+import { appContainer } from '../constants';
+import { Renderable } from '../types/renderable.interface';
+import { Tile } from '../Tile';
+
+export class Board implements Renderable {
+  private readonly tiles: Array<Array<Tile>> = [];
```

Comment:
Most eslint configs I'm used to prevent from using the `Array<>` syntax and force `[]` instead. Are you sure you want to use this syntax?